src/views/FlagsWrapper/index.tsx

```diff
@@ -38,7 +38,8 @@ const UserTargetingWrapper = ({ children }: WrapperProps) => {
             helpFooter: false,
             systemProfile: true,
             systemProfileHiddenFields: false,
-            cedar508Requests: false
+            cedar508Requests: false,
+            notifyMultipleRecipients: true
```

Comment:
This is the default value for the flag that's used when the frontend can't contact LaunchDarkly ([docs](https://docs.launchdarkly.com/sdk/client-side/react/react-web#configuring-the-react-sdk)) - should the value be `false` for the new flag, so we don't accidentally enable it before we're ready?